data/tutorials/gs_00_up_and_running.md

```diff
@@ -11,88 +11,166 @@ date: 2021-05-27T21:07:30-00:00
+- `utop`, an improved REPL,
+- `dune-release` to release code to `opam-repository`, the package base for `opam`.
+
+All these tools can be installed in your current switch (remember that `opam` groups installed packages in independant switches) using the following command:
+
+``` shell
+opam install dune merlin ocaml-lsp-server odoc ocamlformat utop dune-release
 ```
-sh <(curl -sL https://raw.githubusercontent.com/ocaml/opam/master/shell/install.sh)
+
+Now that the tools are installed, it remains to understand how to use them. Most of them will be driven either by the editor or by `dune`, but `utop` is handy to try OCaml or a specific library.
+
+## Up and Running with the platform installer
+
+The platform installer is work in progress, you can follow its development and
+report issues on [the repository](https://github.com/tarides/ocaml-platform-installer/).
+
+Please note that the installer might not work on your system. If that's the
+case, follow the old instructions above, which are still relevant.
+
+The installer is not in any package manager yet but it can be installed using
+this script, this will install both `opam` and `ocaml-platform`:
+
+```shell
+sudo bash < <(curl -sL https://github.com/tarides/ocaml-platform-installer/releases/latest/download/installer.sh)
 ```
 
-### For Windows
+This downloads a script from the web and executes it as root, you are
+encouraged to have a look at what it does first.
+
+Once this step is done, setup the environment with:
 
-OCaml on Windows is still a work in progress. In the meantime, if you only
-need to _run_ OCaml programs on a Windows machine, then the simplest solution is to use the Windows Subsystem for Linux 2 (WSL2). WSL2 is a feature that allows Linux programs to run directly on Windows. WSL2 is substantially easier and faster to use than WSL1. Microsoft has comprehensive installation steps for [setting up WSL2](https://docs.microsoft.com/en-us/windows/wsl/install-win10).
+```shell
+ocaml-platform
+```
 
-On the other hand, if you need Windows binaries, you will have to use the [OCaml for Windows](https://fdopen.github.io/opam-repository-mingw/) installer which comes in 32bit and 64bit versions. This installer gives you Opam and OCaml installations all in one go. It's used from within a Cygwin environment, but the executables produced have no dependency on Cygwin at all. For a more comprehensive update on the state of OCaml on Windows, see the [OCaml on Windows](/docs/ocaml-on-windows) page on the old version of the site.
+This will initialize Opam and install the development tools, which might take
+some time.
 
-## The OCaml Toplevel
+The tools are not installed exactly the same way as with `opam install`: they
+are built in a sandbox so that each tool's dependencies are not installed in
+the same space as your project's dependencies, see [Under the hood](https://github.com/tarides/ocaml-platform-installer#whats-under-the-hood=)
+for more information.
```

Comment:
```suggestion
The tools are not installed exactly the same way as with `opam install`. They
are built in a sandbox so that each tool's dependencies are not installed in
the same space as your project's dependencies, see [Under the Hood](https://github.com/tarides/ocaml-platform-installer#whats-under-the-hood=)
for more information.
```